Utilities/cmlibarchive/libarchive/archive_read_support_format_7zip.c

```diff
@@ -1072,10 +1072,7 @@ ppmd_read(void *p)
 		ssize_t bytes_avail = 0;
 		const uint8_t* data = __archive_read_ahead(a,
 		    zip->ppstream.stream_in+1, &bytes_avail);
-		if(data == NULL) {
-			return (0);
-		}
-		if(bytes_avail < zip->ppstream.stream_in+1) {
+		if(data == NULL || bytes_avail < zip->ppstream.stream_in+1) {
```

Comment:
Should we add a comment here so this doesn't get wiped out by a future merge/insertion?

Reply:
An insertion from kitware? I pushed the same change to libarchive which kitware pulls in eventually, so they'd have the same change. Not sure if I'm misunderstanding the question.